routing/redirect_in_config.rst

```diff
@@ -155,3 +155,76 @@ action:
     Because you are redirecting to a route instead of a path, the required
     option is called ``route`` in the ``redirect()`` action, instead of ``path``
     in the ``urlRedirect()`` action.
+
+Redirecting POST/PUT calls
+--------------------------
+
+As default behaviour of both methods mentioned above results in sending
+response with ``301`` or ``302`` HTTP status codes, the following call will
+be made with use of HTTP request method. But it some scenarios it is
```

Comment:
`use of HTTP request method`
WDYM ? 

`But it some scenarios`
But in some scenarios it's either expected or required that the following call [...]